mpi.h-v3-constified

```diff
@@ -0,0 +1,177 @@
+# A list of functions to intercept via PMPI. This list is meant to be used with MPI version 1.
```

Comment:
should this be "version 3"?

Reply:
Done, with updated comment text in addition.